lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -39,7 +41,9 @@ def scores(self):
         """
         List of all problem scores in the subsection.
         """
-        return [score for score, _ in self.locations_to_weighted_scores.itervalues()]
+        scores = [score for score, _ in self.locations_to_weighted_scores.itervalues()]
+        log.warning("calculated scores property: {0}".format(unicode(scores)))
```

Comment:
Once again, not sure if logging access to this property is needed.  Better to catch issues like this in unit tests.